<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cahoots</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>untitled.txt</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_New"/>
    <addaction name="actionFile_Open"/>
    <addaction name="actionFile_Save"/>
    <addaction name="actionFile_Save_As"/>
    <addaction name="actionFile_Save_A_Copy_As"/>
    <addaction name="actionFile_Save_All"/>
    <addaction name="actionFile_Close"/>
    <addaction name="separator"/>
    <addaction name="actionFile_Print"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuEdit_Syntax_Highlighting">
     <property name="title">
      <string>Syntax Highlighting...</string>
     </property>
     <addaction name="actionEdit_none"/>
     <addaction name="actionEdit_ada"/>
     <addaction name="actionEdit_c"/>
     <addaction name="actionEdit_cpp"/>
     <addaction name="actionEdit_csharp"/>
     <addaction name="actionEdit_css"/>
     <addaction name="actionEdit_flex"/>
     <addaction name="actionEdit_fortran"/>
     <addaction name="actionEdit_glsl"/>
     <addaction name="actionEdit_haxe"/>
     <addaction name="actionEdit_html"/>
     <addaction name="actionEdit_java"/>
     <addaction name="actionEdit_javascript"/>
     <addaction name="actionEdit_latex"/>
     <addaction name="actionEdit_ldap"/>
     <addaction name="actionEdit_logtalk"/>
     <addaction name="actionEdit_lsm"/>
     <addaction name="actionEdit_lua"/>
     <addaction name="actionEdit_m4"/>
     <addaction name="actionEdit_oz"/>
     <addaction name="actionEdit_pascal"/>
     <addaction name="actionEdit_perl"/>
     <addaction name="actionEdit_php"/>
     <addaction name="actionEdit_postscript"/>
     <addaction name="actionEdit_prolog"/>
     <addaction name="actionEdit_python"/>
     <addaction name="actionEdit_ruby"/>
     <addaction name="actionEdit_scala"/>
     <addaction name="actionEdit_sml"/>
     <addaction name="actionEdit_sql"/>
     <addaction name="actionEdit_xml"/>
     <addaction name="actionEdit_xorg"/>
    </widget>
    <addaction name="actionEdit_Undo"/>
    <addaction name="actionEdit_Redo"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Cut"/>
    <addaction name="actionEdit_Copy"/>
    <addaction name="actionEdit_Paste"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Find_All"/>
    <addaction name="actionEdit_Find"/>
    <addaction name="menuEdit_Syntax_Highlighting"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Line_Wrap"/>
    <addaction name="actionView_Hide_Show_Participants"/>
    <addaction name="actionView_Hide_Show_Chat"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="actionText_Shift_Left"/>
    <addaction name="actionText_Shift_Right"/>
    <addaction name="actionText_Comment_Line"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTools_Preview_as_Html"/>
    <addaction name="separator"/>
    <addaction name="actionTools_Announce_Document"/>
    <addaction name="actionTools_Connect_to_Document"/>
    <addaction name="separator"/>
    <addaction name="actionTools_Resynchronize_Document"/>
    <addaction name="separator"/>
    <addaction name="actionTools_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionWindow_Split"/>
    <addaction name="actionWindow_Split_Side_by_Side"/>
    <addaction name="actionWindow_Remove_Split"/>
    <addaction name="separator"/>
    <addaction name="actionWindow_Next_Document"/>
    <addaction name="actionWindow_Previous_Document"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_About_Cahoots"/>
    <addaction name="actionHelp_About_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuText"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile_New"/>
   <addaction name="actionFile_Open"/>
   <addaction name="actionFile_Save"/>
   <addaction name="actionFile_Print"/>
   <addaction name="actionEdit_Undo"/>
   <addaction name="actionEdit_Redo"/>
   <addaction name="actionEdit_Cut"/>
   <addaction name="actionEdit_Copy"/>
   <addaction name="actionEdit_Paste"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFile_Open">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Save_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_New">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Print">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/print.png</normaloff>:/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Cut">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Copy">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Paste">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Find">
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionView_Line_Wrap">
   <property name="text">
    <string>Line Wrap</string>
   </property>
  </action>
  <action name="actionFile_Close">
   <property name="icon">
    <iconset resource="cahootsresc.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_cpp">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionText_Shift_Left">
   <property name="text">
    <string>Shift Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="actionText_Shift_Right">
   <property name="text">
    <string>Shift Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="actionText_Comment_Line">
   <property name="text">
    <string>Un/Comment Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionTools_Preview_as_Html">
   <property name="text">
    <string>Preview as HTML</string>
   </property>
  </action>
  <action name="actionFile_Save_A_Copy_As">
   <property name="text">
    <string>Save A Copy As...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Save_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionTools_Announce_Document">
   <property name="text">
    <string>Announce Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionView_Hide_Show_Participants">
   <property name="text">
    <string>Show/Hide Participants</string>
   </property>
  </action>
  <action name="actionView_Hide_Show_Chat">
   <property name="text">
    <string>Show/Hide Chat</string>
   </property>
  </action>
  <action name="actionTools_Connect_to_Document">
   <property name="text">
    <string>Connect to Document...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionWindow_Split">
   <property name="text">
    <string>Split</string>
   </property>
  </action>
  <action name="actionTools_Resynchronize_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resynchronize Document</string>
   </property>
  </action>
  <action name="actionWindow_Next_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Next Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+]</string>
   </property>
  </action>
  <action name="actionWindow_Previous_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Previous Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+[</string>
   </property>
  </action>
  <action name="actionEdit_none">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionWindow_Remove_Split">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Split</string>
   </property>
  </action>
  <action name="actionWindow_Split_Side_by_Side">
   <property name="text">
    <string>Split Side by Side</string>
   </property>
  </action>
  <action name="actionTools_Preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionEdit_Find_All">
   <property name="text">
    <string>Find All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHelp_About_Cahoots">
   <property name="text">
    <string>About Cahoots</string>
   </property>
  </action>
  <action name="actionHelp_About_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionEdit_ada">
   <property name="text">
    <string>Ada</string>
   </property>
  </action>
  <action name="actionEdit_c">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionEdit_csharp">
   <property name="text">
    <string>C#</string>
   </property>
  </action>
  <action name="actionEdit_haxe">
   <property name="text">
    <string>haXe</string>
   </property>
  </action>
  <action name="actionEdit_css">
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionEdit_flex">
   <property name="text">
    <string>Flex</string>
   </property>
  </action>
  <action name="actionEdit_fortran">
   <property name="text">
    <string>Fortran</string>
   </property>
  </action>
  <action name="actionEdit_glsl">
   <property name="text">
    <string>GLSL</string>
   </property>
  </action>
  <action name="actionEdit_html">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionEdit_java">
   <property name="text">
    <string>Java</string>
   </property>
  </action>
  <action name="actionEdit_javascript">
   <property name="text">
    <string>JavaScript</string>
   </property>
  </action>
  <action name="actionEdit_latex">
   <property name="text">
    <string>LaTeX</string>
   </property>
  </action>
  <action name="actionEdit_ldap">
   <property name="text">
    <string>LDAP</string>
   </property>
  </action>
  <action name="actionEdit_logtalk">
   <property name="text">
    <string>Logtalk</string>
   </property>
  </action>
  <action name="actionEdit_lsm">
   <property name="text">
    <string>LSM</string>
   </property>
  </action>
  <action name="actionEdit_lua">
   <property name="text">
    <string>Lua</string>
   </property>
  </action>
  <action name="actionEdit_m4">
   <property name="text">
    <string>M4</string>
   </property>
  </action>
  <action name="actionEdit_oz">
   <property name="text">
    <string>Oz</string>
   </property>
  </action>
  <action name="actionEdit_pascal">
   <property name="text">
    <string>Pascal</string>
   </property>
  </action>
  <action name="actionEdit_perl">
   <property name="text">
    <string>Perl</string>
   </property>
  </action>
  <action name="actionEdit_php">
   <property name="text">
    <string>PHP</string>
   </property>
  </action>
  <action name="actionEdit_postscript">
   <property name="text">
    <string>PostScript</string>
   </property>
  </action>
  <action name="actionEdit_prolog">
   <property name="text">
    <string>Prolog</string>
   </property>
  </action>
  <action name="actionEdit_python">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
  <action name="actionEdit_ruby">
   <property name="text">
    <string>Ruby</string>
   </property>
  </action>
  <action name="actionEdit_scala">
   <property name="text">
    <string>Scala</string>
   </property>
  </action>
  <action name="actionEdit_sml">
   <property name="text">
    <string>SML</string>
   </property>
  </action>
  <action name="actionEdit_sql">
   <property name="text">
    <string>SQL</string>
   </property>
  </action>
  <action name="actionEdit_xml">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionEdit_xorg">
   <property name="text">
    <string>Xorg</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cahootsresc.qrc"/>
 </resources>
 <connections/>
</ui>
